app/src/main/java/com/android/shelfLife/model/foodFacts/FoodFacts.kt

```diff
@@ -37,7 +37,7 @@ data class FoodFacts(
 }
 
 data class Quantity(
-    val amount: Double, // Amount of the quantity
+    var amount: Double, // Amount of the quantity
```

Comment:
Yeah it good to make this parameter mutable as it allows us to change the amount for the recipes.